pkg/apis/componentconfig/edgecore/v1alpha2/default_kubelet_configuration.go

```diff
@@ -102,4 +98,19 @@ func SetDefaultsKubeletConfiguration(obj *TailoredKubeletConfiguration) {
 	obj.SeccompDefault = utilpointer.BoolPtr(false)
 	obj.MemoryThrottlingFactor = utilpointer.Float64Ptr(configv1beta1.DefaultMemoryThrottlingFactor)
 	obj.RegisterNode = utilpointer.BoolPtr(true)
+
+	if runtime.GOOS == "windows" {
+		obj.EnforceNodeAllocatable = []string{}
+		obj.CgroupDriver = ""
+		obj.CgroupsPerQOS = utilpointer.BoolPtr(false)
+		obj.ResolverConfig = utilpointer.String("")
+		obj.CPUCFSQuota = utilpointer.BoolPtr(false)
+	} else {
+		obj.EnforceNodeAllocatable = DefaultNodeAllocatableEnforcement
+		obj.CgroupDriver = "cgroupfs"
+		obj.CgroupsPerQOS = utilpointer.BoolPtr(true)
+		obj.ResolverConfig = utilpointer.String(kubetypes.ResolvConfDefault)
+		obj.CPUCFSQuota = utilpointer.BoolPtr(true)
+	}
```

Comment:
Could we also move these default values to windows files, same as kubernetes, avoid using `if runtime.GOOS`.

And I'm wondering how k8s to set them for windows?